Martium.GuessTheRiddleGame/Program.cs

```diff
@@ -83,6 +92,13 @@ static int StartGame()
             return progress;
         }
 
+        private static void ShuffleDictionary()
+        {
+            Random shuffle = new Random();
```

Comment:
variables must be named in noun form (daiktavardis) as this is a random instance so you can call it just `random` (shuffle means action)